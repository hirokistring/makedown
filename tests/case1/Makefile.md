# Testcase 1

## `sayhello:`

```sh
@echo "Hello, $(WHO)!"
```

## `saygoodbye:`

> `: sayhello`

```sh
@echo "Bye for now," `date +%Y/%m/%d`
```

```sh
@echo "See you later."
```

## `variables:`

```makefile
WHO = makedown
```
