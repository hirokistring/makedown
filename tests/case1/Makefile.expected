sayhello:
	@echo "Hello, $(WHO)!"
	
saygoodbye: sayhello
saygoodbye:
	@echo "Bye for now," `date +%Y/%m/%d`
	
	@echo "See you later."
	
WHO = makedown


# This makefile is generated by makedown from "Makefile.md"
